worker/README.md

```diff
@@ -73,7 +73,12 @@ Please make sure the RSA public key is uploaded to dispatcher using the public A
 
 ## Example
 
-__note__: your local `PATH_WORKING_DIR` must be group writable (chgrp rwx PATH_WORKING_DIR)
+__note__: your local `PATH_WORKING_DIR` must be group writable and `PATH_WORKING_DIR/sockets` must be world writable:
+
+```
+chmod g+rwx PATH_WORKING_DIR
+mkdir -p -m 0777 PATH_WORKING_DIR/sockets
```

Comment:
I can understand why we have to do this, but I don't like that user have to perform extra step to get things working

Reply:
would you prefer the worker to do it itself?